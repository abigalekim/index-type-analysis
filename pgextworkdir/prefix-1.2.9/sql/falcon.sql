drop table if exists tst;
create table tst ( pref prefix_range );

insert into tst
     select trim(to_char(i, '00000'))
       from generate_series(1, 99999) as i;

select count(*) from tst where pref <@ '55';

create index tst_ix on tst using gist ( pref );

set enable_seqscan = off;

select count(*) from tst where pref <@ '55';

reset enable_seqscan;
