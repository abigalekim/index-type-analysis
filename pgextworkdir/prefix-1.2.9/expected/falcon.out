drop table if exists tst;
NOTICE:  table "tst" does not exist, skipping
create table tst ( pref prefix_range );
insert into tst
     select trim(to_char(i, '00000'))
       from generate_series(1, 99999) as i;
select count(*) from tst where pref <@ '55';
 count 
-------
  1000
(1 row)

create index tst_ix on tst using gist ( pref );
set enable_seqscan = off;
select count(*) from tst where pref <@ '55';
 count 
-------
  1000
(1 row)

reset enable_seqscan;
